dist/angular-chips.js

```diff
@@ -219,11 +217,19 @@
                          */
                         var chipTemplates = iElement.find('chip-tmpl');
                         if (chipTemplates.length > 0 && parseInt(event.target.getAttribute('index')) - 1 === chipTemplates.length)
-                            iElement.find('chip-tmpl')[chipNavigate(37)].focus();
+                            chipNavigate === null ? focusOnChip() : iElement.find('chip-tmpl')[chipNavigate(37)].focus();
                     }
 
                 } else if (event.keyCode === 37 || event.keyCode === 39) {
-                    chipNavigate === null ? focusOnChip() : iElement.find('chip-tmpl')[chipNavigate(event.keyCode)].focus();
+                    if (chipNavigate === null) {
+                      focusOnChip();
+                      if (event.target.nodeName === 'INPUT') {
```

Comment:
Also please make sure that you are covering the new changes in unit test.

Thanks